### DISCUSSION TITLE
Why When I Create A New Note Via Obsidian Uri Templater Not Activate Automatically

### DISCUSSION THREAD

**Discussion by Archie-2021** [SilentVoid13/Templater#655] [Help]
It works fine when I use Templater on plugins like daily note and others but it doesn't trigger when I use Obsidian URI, am I doing something wrong or its a bug?

**Comment by AB1908** [comment]
Are you using Obsidian URIs to create notes?

**Reply by Archie-2021** [reply]
> Are you using Obsidian URIs to create notes?

yes

**Reply by welpdx** [reply]
Using URI to create a new note activates Folder Templates for me.

<details>
           <summary style="font-size:20">Demo</summary>
           <img src="https://i.imgur.com/rEw4uTi.gif" alt="demo" />
</details>


Can you check if you added a template in Settings -> Templater -> Folder Templates? Am I understanding your issue correctly?

Thanks

**Reply by Archie-2021** [reply]
> Using URI to create a new note activates Folder Templates for me.
> 
> Demo
> Can you check if you added a template in Settings -> Templater -> Folder Templates? Am I understanding your issue correctly?
> 
> Thanks

Yes, I have one but for a specific folder unrelated to this one.
PS. I tested it and it works if I turn it off, but the setting I have is for a different folder, how can I fix it?

**Reply by welpdx** [reply]
Ok, I am not sure what you mean but for me, Obsidian URI doesn't work if I try to create a note where the `Template folder location` is. 
<details>
           <summary style="font-size:20">Issue</summary>
           <img src="https://i.imgur.com/F2esBjI.gif" alt="demo" />
</details>

Other than that, it works ok for me. 

Can you elaborate on what your problem is? Do you want to use multiple Folder templates?

Thanks

**Reply by Archie-2021** [reply]
> Ok, I am not sure what you mean but for me, Obsidian URI doesn't work if I try to create a note where the `Template folder location` is.
> 
> Issue
> Other than that, it works ok for me.
> 
> Can you elaborate on what your problem is? Do you want to use multiple Folder templates?
> 
> Thanks

I can't get you too, look: I have a Folder creation on and only one setting for  Folder A with Template A for it, as soon as I had that template stopped to auto-activate on Folder B which is not related to Folder A (nested under or reverse). Now I can't see why I can't have both (auto activation and special template for folders). Am I doing something wrong or its a bug. Also I should emphasis again that I just one one settings (one template mapping to one folder something totally normal)

Thank you for your help
